packages/next-app/src/components/Plot.tsx

```diff
@@ -0,0 +1,47 @@
+import { amberDarkA, blueDarkA, grayDarkA } from "@radix-ui/colors";
+import { merge } from "lodash";
+import PlotRoot, { PlotParams } from "react-plotly.js";
+
+export const defaultAxisLayout = {
+  gridcolor: grayDarkA.grayA10,
+  linecolor: grayDarkA.grayA12,
+  linewidth: 0.5,
+  automargin: true,
+  zerolinecolor: grayDarkA.grayA10,
+};
+
+export const defaultPlotProps = {
+  className: "w-full h-full",
+  useResizeHandler: true,
+  layout: {
+    plot_bgcolor: blueDarkA.blueA1,
+    paper_bgcolor: blueDarkA.blueA1,
+    font: {
+      color: grayDarkA.grayA12,
+      family: "Inter",
+    },
+    xaxis: JSON.parse(JSON.stringify(defaultAxisLayout)),
+    yaxis: JSON.parse(JSON.stringify(defaultAxisLayout)),
+    modebar: {
+      bgcolor: blueDarkA.blueA1,
+      color: grayDarkA.grayA12,
+      activecolor: grayDarkA.grayA12,
+      orientation: "h",
+    },
+    colorway: [blueDarkA.blueA9, amberDarkA.amberA9],
+  },
+  config: {
+    displaylogo: false,
+    showAxisRangeEntryBoxes: false,
+    showSendToCloud: false,
+    showEditInChartStudio: false,
+    showLink: false,
+    watermark: false,
+  },
+  revision: 0,
+};
+
+export default function Plot(props: PlotParams) {
+  const plotProps = merge(JSON.parse(JSON.stringify(defaultPlotProps)), props);
```

Comment:
this is very odd and prone to WTFs by newcomers -- if we must do it please leave a comment here so whomever understands why this is needed

Reply:
This is needed bc the merge changes the first object (defaultPlotProps) with the function result. So, I am using the json.parse/json.stringify to deep copy the defaultPlotProps